doc/style-guide.md

```diff
@@ -44,3 +45,52 @@ quaility.
     character limit.
 
 **[top](#table-of-contents)**
+
+## Expressions
+
+  - Expressions in return statements should be surrounded by parenthesis for clarity.
```

Comment:
Is there better wording to indicate we mean expressions that aren't just a value? Technically "a" is an expression, right? Or does javascript have different terminology for that?